plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherExtension.kt

```diff
@@ -82,4 +83,10 @@ open class PlayPublisherExtension : AccountConfig by PlayAccountConfigExtension(
                 _track = TrackType.ROLLOUT
             releaseTrackSet = true
         }
+
+    /**
+     * Optional folder to find APK files. Only for use when the APK is built by an external process
+     */
+    var overrideBuildOutput: File? = null
```

Comment:
This property name is confusing. It makes it sound like it's _our_ output. `apkInputFolder` makes more sense.

Reply:
I think it should be more generic than that. But agree on the `input` vs. `output`